code/modules/surgery/bones.dm

```diff
@@ -154,3 +156,48 @@
 	var/datum/organ/external/affected = target.get_organ(target_zone)
 	user.visible_message("<span class='warning'>[user]'s hand slips, smearing [tool] in the incision in [target]'s [affected.display_name]!</span>" , \
 	"<span class='warning'>Your hand slips, smearing [tool] in the incision in [target]'s [affected.display_name]!</span>")
+
+
+
+//////BONE MENDER/////////
+/datum/surgery_step/bone_mender
+	allowed_tools = list(
+		/obj/item/weapon/bonesetter/bone_mender = 100,
+		)
+
+	priority = 0.1 //so it trys to do this before /glue_bone
+
+	min_duration = 80
+	max_duration = 90
+
+/datum/surgery_step/bone_mender/can_use(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+	var/datum/organ/external/affected = target.get_organ(target_zone)
+	return (affected.open >= 2 || (target.species.anatomy_flags & NO_SKIN)) && affected.stage <= 5
+
+/datum/surgery_step/bone_mender/begin_step(mob/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+	var/datum/organ/external/affected = target.get_organ(target_zone)
+	if (affected.stage <= 5)
+		user.visible_message("[user] starts grasping the damaged bone edges in [target]'s [affected.display_name] with \the [tool]." , \
```

Comment:
`\The [user]`, `\the [target]`

Reply:
Pretty sure thats correct as is. "Bob starts grasping the damaged bone edges in steve's leg with the bone mender."

Reply:
"unknown starts grasping the damaged bone edges in unknown's leg with the bonemender"

Reply:
I will do some testing when ai get the chance

Reply:
There's no difference
I keep telling people this
The `\the` macros don't do anything because BYOND automatically inserts them in all places they are allowed to be
We just say to use them for style reasons
Personally I think this is fine how it is EXCEPT the other line I've commented on below, because everywhere it's not included will usually be a proper noun, and everywhere it is included usually won't. It'll still behave properly regardless of whether or not the macros are there, but it reads more smoothly this way.